lldb/source/Plugins/TypeSystem/Swift/SwiftASTContext.h

```diff
@@ -1037,8 +1014,45 @@ class SwiftASTContextForExpressions : public SwiftASTContext {
 
   void ModulesDidLoad(ModuleList &module_list);
 
-private:
-  std::unique_ptr<SwiftPersistentExpressionState> m_persistent_state_up;
+  typedef llvm::StringMap<swift::AttributedImport<swift::ImportedModule>>
+      HandLoadedModuleSet;
+
+  // This just adds this module to the list of hand-loaded modules, it doesn't
+  // actually load it.
```

Comment:
It doesn't only _add_ the list of modules, it could overwrites an existing value, if any.